.github/workflows/test-github-backport-active.yml

```diff
@@ -0,0 +1,62 @@
+name: test-github-backport-active
+
+on:
+  merge_group: ~
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: 'Dry run'
+        required: true
+        default: true
+        type: boolean
+  push:
+    paths:
+      - '.github/workflows/test-github-backport-active.yml'
+      - 'github/backport-active/**'
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  test-backport-action:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
we don't use `v3` is too old. If yous e use any other actions please use the latest major version, that's only when using `actions/*`. If using any other third-party action please use a pinned version - that's not in this case